modules/core/src/lib/constants.js

```diff
@@ -42,3 +43,13 @@ export const COORDINATE_SYSTEM = {
   // Positions and distances are not transformed: [x, y, z] in unit coordinates
   IDENTITY: 0.0
 };
+
+export const SHADER_COORDINATE_SYSTEM = {
```

Comment:
The point of this split is that `SHADER_COORDINATE_SYSTEM` should not be a "global" constant, this should be defined locally by the project module system.

In fact we should probably name it `PROJECT_COORDINATE_SYSTEM`.

There should be one such set of definitions in JS, and one in GLSL (a set of `#define`s) that carefully match each other